pyrobosim/examples/demo_astar.py

```diff
@@ -0,0 +1,38 @@
+import os
```

Comment:
This demo doesn't work because of circular imports. I think I broke some of the ordering in imports from the last PR, where now import order matters.

```
  File "/home/sebastian/personal/pyrobosim_ws/src/pyrobosim/pyrobosim/pyrobosim/core/robot.py", line 11, in <module>
    from ..utils.knowledge import resolve_to_object
ImportError: cannot import name 'resolve_to_object' from partially initialized module 'pyrobosim.utils.knowledge' (most likely due to a circular import) (/home/sebastian/personal/pyrobosim_ws/src/pyrobosim/pyrobosim/pyrobosim/utils/knowledge.py)
```